draft-js-plugins-editor/CHANGELOG.md

```diff
@@ -14,6 +14,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - `decorators` don't decorate plugins anymore.
 
+### Changed
+
+- *Breaking change* The `plugins` and `decorators` editor props need to be an Immutable.List()
```

Comment:
@MarkNijhof Totally agree! The trouble we face is that the decorator is actually part of the EditorState and not passed to the Editor in Draft.js.
see https://github.com/facebook/draft-js/blob/master/examples/tweet/tweet.html#L49

If I'm not mistaken you can't compare the decorators by reference, because they are not an immutable object. We could also change that, but then it this would diverge from Draft.js and be inconsistent with the rest of the plugin architecture. It's a tricky edgecase.